src/components/recipes/create/RecipesCreate.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useState } from 'react';
+import { FormEvent, FC } from 'react';
+import { useCreateRecipe } from 'hooks/recipes';
+import { RequestParamsTypeCreateRecipe } from 'types/api';
+import StringsHelper from 'helpers/StringsHelper';
+import { Button, Card, Container, Input } from 'styled/UIComponents';
+import { useNavigate } from 'react-router-dom';
+
+const useHandleSubmit = (params: RequestParamsTypeCreateRecipe) => {
+    const createRecipe = useCreateRecipe();
+    const navigate = useNavigate();
+
+    const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        createRecipe.mutate(params, {
+            onSuccess: () => {
+                navigate('/recipes');
+                return;
+            },
+        });
+    };
+
+    return [handleSubmit] as const;
+};
+
+const RecipesCreate: FC = () => {
+    const [name, setName] = useState('');
+    const [authorId, setAuthorId] = useState('');
+    const [ingredients, setIngredients] = useState('');
```

Comment:
You might not want to in this case, but another nice way to do this is to store the form data as an object in the state, e.g.

```ts
const [formData, setFormData] = useState({
   name: '',
   authorId: '',
   ingredients: []
})
```

Means you can pass the form data around a bit more nicely.